src/plugin/tasks/serverless/snippets/serverless/triggers/api/${trigger}.yml

```diff
@@ -0,0 +1,16 @@
+resources:
+  Resources:
+    Trigger${trigger|TITLE}LambdaResource:
+      DependsOn:
+        - PostDashdeployLambdaFunction
+      Type: 'Custom::LambdaDependency'
+      Properties:
+        ServiceToken:
+          'Fn::GetAtt':
+            - PostDashdeployLambdaFunction
+            - Arn
+        custom: '${self:custom.randomDeployHash}'
+
+functions:
+  ${trigger|CAMEL}Fn:
+    handler: lib/handler.${trigger|CAMEL}Fn
```

Comment:
I do not understand this `DependsOn` syntax to make it dynamic.

Are we making `randomDeployHash` dynamic and creating a file for every instance of a trigger??